lib/theme_check/analyzer.rb

```diff
@@ -53,16 +70,23 @@ def analyze_files(files)
         # Call all checks that run on the whole theme
         liquid_visitor = LiquidVisitor.new(@liquid_checks.whole_theme, @disabled_checks)
         html_visitor = HtmlVisitor.new(@html_checks.whole_theme)
-        @theme.liquid.each do |liquid_file|
+        total = total_file_count + files.size
```

Comment:
Hum... this is weird. `total_file_count` should already include `files.size`. Are you sure this shouldn't simply be `total_file_count`?

Reply:
Yeah that part is weird. It's because we're actually running theme check on all files for all the non global checks, and then on the files themselves. So we're running the files twice.